src/main/scala/com/kinja/config/safeConfig.scala

```diff
@@ -100,8 +95,7 @@ object safeConfig {
 
       // Replaces references between config values with the private name.
       val transformer = new Transformer {
-        import collection.mutable.Stack
-        val stack : Stack[Map[TermName, TermName]] = Stack(configValues.map {
+        var stack : List[Map[TermName, TermName]] = List(configValues.map {
```

Comment:
This is exactly what the Scala 2.12 compiler recommended:
```
class Stack in package mutable is deprecated (since 2.12.0): Stack is an inelegant and potentially poorly-performing wrapper around List. Use a List assigned to a var instead.
```

Reply:
No arguing here. I wonder if we really need mutable variables though.

Reply:
Rewriting macros for 2.13 will suck anyways, so let's address that in the future :)

Reply:
It surely will.